ogticrd/covid-safe-paths
# fixing news header

#### Description: 


I modified the header of the screen news and the form to get the image of every news
#### Linked issues:


fixes: [#93](https://trello.com/c/OLrI96bs/93-modify-the-header-in-news-screen-to-have-a-better-view-of-the-list)
#### Screenshots:


![image](https://user-images.githubusercontent.com/43874147/83190567-9def5080-a100-11ea-9f38-d726f323cde5.png)

#### How to test:


- `git pull`
- `git checkout Fix/93-header-news-screen`
- Head over to screen news and check how does it look
